config/routes.rb

```diff
@@ -186,6 +186,11 @@
       resources :partnerships, only: :index
       resource :programme, only: %i[edit], controller: "choose_programme"
 
+      resources :participants, only: %i[index show update], constraints: ->(_request) { FeatureFlag.active?(:induction_tutor_manage_participants) } do
+        get :edit_details, path: "edit-details"
+        get :edit_mentor, path: "edit-mentor"
+      end
```

Comment:
I think I'd avoid using FeatureFlags in the routes. This file is only loaded once so enabling the feature would require server reset.